src/meta/src/hummock/manager/mod.rs

```diff
@@ -762,11 +767,15 @@ where
             .generate::<{ IdCategory::HummockCompactionTask }>()
             .await?;
 
-        let group_config = self
+        let group_config = match self
             .compaction_group_manager
             .read()
             .await
-            .get_compaction_group_config(compaction_group_id);
+            .try_get_compaction_group_config(compaction_group_id)
```

Comment:
If we use get_or_insert_compaction_group_config to initialize cg, then why can't we get config?

Reply:
When the last table of a compaction group is deleted, the compaction group (and its config) is destroyed as well. Then a compaction task for this gorup may come later and cannot find conifg.

Reply:
Thanks , need some comments ?